Wikipedia/Code/WMFSettingsViewController.m

```diff
@@ -158,7 +158,7 @@ - (void)tableView:(UITableView*)tableView didSelectRowAtIndexPath:(NSIndexPath*)
             [self wmf_openExternalUrl:[NSURL URLWithString:WMFSettingsURLZeroFAQ]];
             break;
         case WMFSettingsMenuItemType_RateApp:
-            [self wmf_openExternalUrl:[NSURL URLWithString:WMFSettingsURLRate]];
+            [[UIApplication sharedApplication] openURL:[NSURL URLWithString:WMFSettingsURLRate]]; //always exit the app
```

Comment:
sorry but, technically yes we do. so i would recommend modifying `wmf_openExternalURL` by adding another `BOOL` parameter which tells the logic whether it should open in app or Safari